docs/standard/microservices-architecture/architect-microservice-container-applications/resilient-high-availability-microservices.md

```diff
@@ -37,37 +42,36 @@ A microservice-based application should not try to store the output stream of ev
 
 When you create a microservice-based application, you need to deal with complexity. Of course, a single microservice is simple to deal with, but dozens or hundreds of types and thousands of instances of microservices is a complex problem. It is not just about building your microservice architecture—you also need high availability, addressability, resiliency, health, and diagnostics if you intend to have a stable and cohesive system.
 
-![](./media/image22.png)
+![Orchestrators supply a support platform for running your microservices.](./media/image22.png)
 
-**Figure 4-22**. A Microservice Platform is fundamental for an application’s health management
+**Figure 4-22**. A Microservice Platform is fundamental for an application's health management
 
 The complex problems shown in Figure 4-22 are very hard to solve by yourself. Development teams should focus on solving business problems and building custom applications with microservice-based approaches. They should not focus on solving complex infrastructure problems; if they did, the cost of any microservice-based application would be huge. Therefore, there are microservice-oriented platforms, referred to as orchestrators or microservice clusters, that try to solve the hard problems of building and running a service and using infrastructure resources efficiently. This reduces the complexities of building applications that use a microservices approach.
 
 Different orchestrators might sound similar, but the diagnostics and health checks offered by each of them differ in features and state of maturity, sometimes depending on the OS platform, as explained in the next section.
 
 ## Additional resources
 
--   **The Twelve-Factor App. XI. Logs: Treat logs as event streams**
-    [*https://12factor.net/logs*](https://12factor.net/logs)
-
--   **Microsoft Diagnostic EventFlow Library.** GitHub repo.
+-   **The Twelve-Factor App. XI. Logs: Treat logs as event streams**  
+    *https://12factor.net/logs*
 
-    [*https://github.com/Azure/diagnostics-eventflow*](https://github.com/Azure/diagnostics-eventflow)
+-   **Microsoft Diagnostic EventFlow Library.** GitHub repo.  
+    *https://github.com/Azure/diagnostics-eventflow*
 
--   **What is Azure Diagnostics**
-    [*https://docs.microsoft.com/azure/azure-diagnostics*](https://docs.microsoft.com/azure/azure-diagnostics)
+-   **What is Azure Diagnostics**  
```

Comment:
we're not relying on two empty spaces at the end to create a new line. If you need a new line, add one.

Reply:
is it ok to add <br/>? it achieves the same as the two spaces.

Reply:
I mean, to have the link begin in the new line, but not creating a new bullet. It's a more consistent look and easier to read:
![image](https://user-images.githubusercontent.com/7357806/45874021-ba63b300-bd8b-11e8-9f2c-7e1219867116.png)

Reply:
added `<br/>` to keep the arrangement, i.e. the name and the link in the same bullet.